src/main/java/uk/gov/hmcts/ccd/domain/model/search/global/SearchCriteriaResponseTEMP.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.ccd.domain.model.search.global;
```

Comment:
temp classes for the controller use only - to be removed once testing is complete